### DISCUSSION TITLE
Ask your questions in GitHub Discussions!

### DISCUSSION THREAD

**Discussion by MatthiasValvekens** [MatthiasValvekens/pyHanko#41] [Announcements]
Hi everyone,

I'd like to try something new: starting now, I'm designating this space as the forum for asking questions & general public discussion about pyHanko. GitHub discussions have been around for a while now, we might as well use 'em.

If you're facing an issue that's not a bug or an enhancement request, **consider starting a discussion here** instead of opening an issue on the bug tracker. Any bug reports or feature requests that surface over the course of the discussion can then be turned into issues separately.

Additionally, if you notice comments on an existing issue veering off-topic, consider taking the conversation into this discussion space.

Thanks!

**Comment by roccolocko** [comment]
Hello Matthias,

Is there a way to remove the the black border from the signatures when using the visual stamping feature?

Thank you!

**Reply by MatthiasValvekens** [reply]
Hi @roccolocko, would you mind creating a separate thread with your question? That helps to keep things organised and searchable. Thanks!
